packages/registry/src/Registry.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  Attribute,
+  FetchAttributeInput,
+  FetchVersionDropAttributeInput,
+  FetchAttributesInput,
+  VersionedAttribute,
+} from './types';
+import { PaginatedResult } from './utils/types';
+import { RegisrtyProvider } from './ports/RegistryProvider';
+
+export class Registry {
```

Comment:
Why again we have a class that does nothing else than call the provider? What does this class do?